### PR TITLE
RDAPI-24970 Filter requirements

### PR DESCRIPTION
**Pull request by dgascoin-Axway**
Thank you for your contribution to the Axway-Open-Docs repo.

## Describe the changes

Add requirements to the throttling filter based on https://jira.axway.com/browse/RDAPI-24970

## Deploy preview link

Please add the deploy preview link to the **specific page** that you've changed.

https://deploy-preview-2056--axway-open-docs.netlify.app/docs/apim_policydev/apigw_polref/content_max_messages/

## Checklist for contributors

Before submitting this PR, please make sure:

* [x] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [x] You have signed the [Axway CLA](https://cla.axway.com/)
* [x] You have verified the technical accuracy of your change
* [x] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [x] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)
* [x] You have verified that all status checks have passed

_Put an x in the boxes that apply. This is simply a reminder of what we are going to look for before merging your change._

### CODE REVIEW

File: content/en/docs/apim_policydev/apigw_polref/content_max_messages.md [hunk 1]
```diff
@@ -5,6 +5,9 @@
   "date": "2019-10-17",
   "description": "Use the Throttling filter to rate limit calls to a back-end service."
 }
+
+{{< alert title="Note" color="primary" >}}This filter requires Cassandra and API Manager or a KPS table to work.{{< /alert >}}
```
Comment (kmercierwhite) [inline content/en/docs/apim_policydev/apigw_polref/content_max_messages.md:9]: having "and" and "or" in the same sentence is gramatically confusing, so I suggest

This filter requires a KPS tabled using Cassandra to work. This includes API Manager KPS tables.

Comment (dgascoin-Axway) [reply content/en/docs/apim_policydev/apigw_polref/content_max_messages.md]: Sorry for my bad English, updated thanks !

File: content/en/docs/apim_policydev/apigw_polref/content_max_messages.md [hunk 2]
```diff
@@ -6,10 +6,10 @@
   "description": "Use the Throttling filter to rate limit calls to a back-end service."
 }
 
-{{< alert title="Note" color="primary" >}}This filter requires a KPS table using Cassandra to work. This includes API Manager KPS tables.{{< /alert >}}
-
 The **Throttling** filter enables you to limit the number of requests that pass through an API Gateway in a specified time period. This enables you to enforce a specified message quota or *rate limit* on a client application, and to protect a back-end service from message flooding.
 
+{{< alert title="Note" color="primary" >}}This filter requires a [Key Property Store (KPS)](/docs/apim_policydev/apigw_kps/get_started/) table, which can be, for example, an API Manager KPS table.{{< /alert >}}
```
Comment (andreamussap) [inline content/en/docs/apim_policydev/apigw_polref/content_max_messages.md:11]: @kmercierwhite is this ok, can I merge the PR?

Comment (kmercierwhite) [reply content/en/docs/apim_policydev/apigw_polref/content_max_messages.md:11]: Looks good to me!

**Review by andreamussap — APPROVED** [review]

**Pull request merged**